### DISCUSSION TITLE
白群辉ds220+  docker cloudsync  同步完后 过几天就提示异常：同步的文件夹不存在

### DISCUSSION THREAD

**Discussion by umbrellax** [messense/aliyundrive-webdav#549] [Q&A]
### 问题描述

白群辉ds220+  docker运行 cloudsync  同步完后 过几天就提示异常：同步的文件夹不存在
本地路径和远程路径都能打开都没问题
删除重新添加项目后可短暂恢复正常

[图片显示](http://tc.nobb.cc/imgs/2022/08/ef907ed7dc7cf2be.jpg)

### 重现步骤

1.正常添加同步路径
2.过几天自动提示异常

### 版本

1.6.8

### 运行平台

群辉docker

### refresh token 获取方式

网页

### 日志

日志没问题

**Comment by tmzg0000** [comment]
我也有这个问题，我用这个网页的方法 恢复。https://www.encrypted.at/fix-for-synology-cloud-sync-sync-folder-does-not-exist/
但是怎么避免发生我不清楚。希望高手解决

**Reply by clocktowerc** [reply]
感谢链接，请问有研究出发生的原因以及避免的办法吗？
